js/provider.js

```diff
@@ -0,0 +1,329 @@
+const showMsg = (id ,errorMsg ,status ) =>{
+    let place = document.getElementById(`${id}`);
+    let msgField = place.querySelector(`#msg`);
+    place != null ? place.classList.toggle(`${status}`) : '' ;
+    msgField != null ? msgField.innerHTML = errorMsg : '' ;
+    setTimeout(() => {
+        msgField != null ?  msgField.innerHTML = '': '' ;
+        place.classList.toggle(`${status}`) ;
+    } ,2000);
+}
+const showNotification = (title = '',msg = '',errorType = '' , lastIn = null) =>{
+  
+    let place = document.getElementById(`notification`);
+    let msgTitleField = place.querySelector(`#error-title`);
+    let notInfo = place.querySelector(`#not-info`);
+    msgTitleField.innerHTML =  title;
+    notInfo.innerHTML = msg  ;
+    place.classList.add(`${errorType}-msg`);
+    if(lastIn == null){        
+        setTimeout(() => {
+            msgTitleField != null ?  msgTitleField.innerHTML = '': '' ;
+            notInfo != null ?  notInfo.innerHTML = '': '' ;
+            place.classList.remove(`${errorType}-msg`);
+        } ,4000);
+    }
+    else{
+        // setTimeout(() => {
+        //     msgTitleField != null ?  msgTitleField.innerHTML = '': '' ;
+        //     notInfo != null ?  notInfo.innerHTML = '': '' ;
+        //     place.classList.remove(`${errorType}-msg`);
+        // } ,lastIn);
```

Comment:
remove this comments if you're not using it

Reply:
also this are in the second pr